doc/cookbook/source/examples/distance/cosine.rst

```diff
@@ -0,0 +1,39 @@
+==================
+Cosine	 Distance
+==================
+
+The Cosine distance for real valued features is the similarity between two vectors by finding their angle.
+
+.. math::
+
+    1-\frac{{\bf x^\top x'}}{\Vert \bf{x}\Vert_2 \Vert \bf{x'}\Vert_2 }
+
+where where :math:`\Vert \cdot\Vert_2` is the Euclidean norm, :math:`\bf x` and :math:`\bf x'` are :math:`d` dimensional feature vectors.
```

Comment:
now you can actually remove the "d  dimensional" -- d is never used anywhere. Just say in the first sentence

"The Cosine distance for real valued features x and x' is the similarity as measured by their angle." And then remove the ":math:`\bf x` and :math:`\bf x'` are :math:`d` dimensional feature vectors."